docs/PartRendering.md

```diff
@@ -0,0 +1,52 @@
+# Rendering of parts
+
+## Server side and client side rendering
+
+Due to a bug in react4xp we had to render all parts on the client side in order for them to work. The bug has now been fixed, and we can properly set up serverside rendering for all parts.
+
+## Removing clientside rendering
+
+Most of our parts use clientRender when returning body and pageContributions. This needs to be deleted:
```

Comment:
Suggestion 2 (rephrased "Removing clientside rendering" paragraph): 
React4xp components are serverside rendered by default, so in order to disable clientside rendering we have to remove the clientRender property in the renderBody and renderPageContributions calls from most of our parts: